components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -249,11 +409,15 @@ public static OMElement buildOMElement(InputStream inputStream) throws Exception
     public List<AppCallTypeDTO> getAppApiCallType(String subscriberName, String groupId, String fromDate, String toDate, int limit)
             throws APIMgtUsageQueryServiceClientException {
 
-        OMElement omElement = this.queryBetweenTwoDays(
-                APIUsageStatisticsClientConstants.API_Resource_Path_USAGE_SUMMARY, fromDate, toDate);
-        Collection<AppCallType> usageData = getCallTypeUsageData(omElement);
-
         List<String> subscriberApps = getAppsbySubscriber(subscriberName, groupId);
+        String[] stringArray = subscriberApps.toArray(new String[subscriberApps.size()]);
```

Comment:
Do we need to create a array from the list here. I think buildKeySetString(stringArray); can take the list and do the same logic?